wgpu/src/backend/wgpu_core.rs

```diff
@@ -2994,6 +3023,62 @@ fn default_error_handler(err: crate::Error) {
     panic!("wgpu error: {err}\n");
 }
 
+impl From<&CreateShaderModuleError> for CompilationInfo {
```

Comment:
The CI fails, because there's a configuration where need this trait in `wgpu/src/backend/webgpu.rs`, but `#[cfg(wgpu_core)]` doesn't apply. So it doesn't get compiled, and Rust rightfully throws an error.

Any suggestions on where to move it?